contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -282,6 +282,10 @@ static int
 change_key(pam_handle_t *pamh, const char *ds_name,
     const char *passphrase)
 {
+	uint64_t keyformat = ZFS_KEYFORMAT_NONE;
+	FILE *file = NULL;
+	char prop_keylocation[MAXNAMELEN];
```

Comment:
nit: This could be moved in the `if (keyformat != ZFS_KEYFORMAT_NONE) { }` block to limit its scope.

Reply:
will fix in next revision, thanks.